receiver/fluentforwardreceiver/config.go

```diff
@@ -24,4 +28,32 @@ type Config struct {
 	// of the form `<ip addr>:<port>` (TCP) or `unix://<socket_path>` (Unix
 	// domain socket).
 	ListenAddress string `mapstructure:"endpoint"`
+
+	Mappings MappingWhitelist `mapstructure:"mappings"`
+
+	BodyAsString bool `mapstructure:"body_as_string"`
+}
+
+func (c *Config) validate() error {
+	if c.ListenAddress == "" {
+		return errors.New("`endpoint` not specified")
+	}
+
+	/*if c.BodyAsString == nil {
+		c.BodyAsString = true
+	}*/
+
+	if len(c.Mappings.Body) == 0 {
+		c.Mappings.Body = []string{"log", "message"}
```

Comment:
Default value for retrocompatibility.
Both log or message will be considered as "body" value